workflow/rules/trimming.smk

```diff
@@ -53,39 +53,61 @@ rule cutadapt_se:
     wrapper:
         "0.42.0/bio/cutadapt/se"
 
-
-rule trimmomatic_se:
+rule pipe_ptrimmer_se:
     input:
         "results/trimmed/adapters/{sample}/{unit}.single.fastq.gz"
     output:
-        "results/trimmed/primers/{sample}/{unit}.single.fastq.gz",
+        pipe("results/trimmed/adapters/{sample}/{unit}.single_R1.fastq.gz")
+    threads: 0 # this does not need CPU
+    shell:
+        "cat {input} > {output} 2> {log}"
+
+
+rule pipe_ptrimmer_pe:
+    input:
+        "results/trimmed/adapters/{sample}/{unit}.{read}.fastq.gz"
```

Comment:
Let us rather rename the files in the pipeline into R1/R2. This way, we can omit these two rules.

Reply:
I tried this at first but I was not able to resolve this as ptrimmer needs the "_R" syntax even for single end mode why is why we also must rename "single" into "_R1".
This way it was not possible to distingish between paired end and single end rules.